src/core/storage/sqliteStorage/sqliteSession.ts

```diff
@@ -44,6 +47,20 @@ class SqliteSession {
 
   async open(storageName: string): Promise<void> {
     const connection = new SQLiteConnection(CapacitorSQLite);
+
+    const platform = Capacitor.getPlatform();
+    const isPlatformEncryption = platform !== "web";
+    const isEncryptInConfig = (await connection.isInConfigEncryption()).result;
+    const isEncryption = isPlatformEncryption && isEncryptInConfig;
+    if (isEncryption) {
+      const isSetSecret = (await connection.isSecretStored()).result;
+      if (!isSetSecret) {
+        const newBran = (await Agent.agent.getBranAndMnemonic()).bran;
```

Comment:
You can just use `randomPasscode()` from Signify, and not bother with all the BIP-39 conversions in the background.

You don't really need the `substring(0, 21)` that is used elsewhere. Signify was generating 22 chars for legacy reasons but now does 21. Only first 21 were ever used by KERI.

Reply:
done!